widget/class-widget.php

```diff
@@ -97,8 +112,8 @@ public function widget( $args, $instance ) {
 
       echo $args['before_widget'];
 
-      if ( ! empty( $instance['title'] ) ) {
-         echo $args['before_title'] . apply_filters( 'widget_title', $instance['title'] ) . $args['after_title'];
+      if ( ! empty( $instance['title'] ) ) { ?>
+         <a href="<?php echo $allOppsPageLink; ?>"><?php echo $args['before_title'] . apply_filters( 'widget_title', $instance['title'] ) . $args['after_title']; ?></a> <?php
```

Comment:
We need to check if there is a page to link to first before we add this. It's possible an admin would create volunteer opportunities before adding one of the shortcodes to a page.